.github/workflows/ci.yml

```diff
@@ -544,6 +544,7 @@ jobs:
             kubectl apply -f config/samples/simple.ingress.yaml
           else
             kubectl apply -f config/samples/simple.yaml
+            kubectl patch pulp example-pulp --type=merge -p '{"spec":{"api":{"strategy":{"type":"Recreate"}}}}'
```

Comment:
we need to define the `Recreate` strategy specifically to upgrade's workflow due to an issue with pod redeploy and port-forward command.